queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Excellent work 👍🏻 , however, make sure you really understand what the question says. ''you should list the name of all animals born between 2016 and 2019''  that means it should list from  `2016-01-01` until `2019-31-12` instead of `2019-01-01` that case you will be sure to list the name of all animals born in 2019, Kindly change this 😄

Reply:
Thanks Ben. The question is not clear. I thought about the meaning of between in this context and this was my result. If I understood it wrong, then sure I will change it. Thanks for your feedback.

Reply:
Great

Reply:
This issue is well resolved! ✔️